smartapps/juano2310/jawbone-up-connect.src/jawbone-up-connect.groovy

```diff
@@ -349,67 +352,49 @@ def uninstalled() {
 }
 
 def pollChild(childDevice) {
-    def member = state.member
-    generatePollingEvents (member, childDevice)
+		def params = [
+      uri: 'https://jawbone.com',
+      path: '/nudge/api/users/@me/goals',
+      headers: ["Authorization": "Bearer ${state.JawboneAccessToken}" ]
+    ]
+    asynchttp_v1.get('responseGoals', params, childDevice)
+
+		def params = [
+			uri: 'https://jawbone.com',
+			path: '/nudge/api/users/@me/moves',
+			headers: ["Authorization": "Bearer ${state.JawboneAccessToken}" ]
+		]
+		asynchttp_v1.get('responseMoves', params, childDevice)
```

Comment:
Is this second http request dependent on the first in any way? Since the request is made asynchronously, the second request may be made before the response comes back from the first. If the second request should only be made when the first request completes, consider moving the second request in to the response handler of the first.